debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/AbstractReader.java

```diff
@@ -38,7 +36,7 @@
     protected final Logger logger = LoggerFactory.getLogger(getClass());
     private final String name;
     protected final MySqlTaskContext context;
-    protected final MySqlJdbcContext connectionContext;
+    protected final MySqlConnection connectionContext;
```

Comment:
So, @jpechane  this is where MySqlJdbcContext has been renamed to MySqlConnection. This has happened on a few occasions below.